oolab/src/main/java/agh/ics/oop/RectangularMap.java

```diff
@@ -0,0 +1,57 @@
+package agh.ics.oop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RectangularMap implements IWorldMap{
+
+    private List<Animal> animals = new ArrayList<>();
+    private static final Vector2d LOWER_BOUND = new Vector2d(0,0);
+    private static Vector2d UPPER_BOUND; /* Yup, I know it's not a constant value,
```

Comment:
ok, ale jak Pan będzie pracował w jakimś większym zespole to dla innych będzie to mylące ;P

Reply:
Fair point